Support Azure Migrate Assessment Project (s) (#426)
inspec/inspec-azure: docs/resources/azure_migrate_assessment_project.md

| type                          | Type of the object. `Microsoft.Migrate/assessmentProjects`       |
| eTag                          | For optimistic concurrency control.                              |
| properties                    | Properties of the project.                                       |
| location                      | Azure location in which project is created.                      |
| properties.assessmentSolutionId | Assessment solution ARM id tracked by `Microsoft.Migrate/migrateProjects`.|
| properties.customerStorageAccountArmId| The ARM ID of the storage account used for interactions when public access is disabled.|
| properties.privateEndpointConnections | The list of private endpoint connections to the project. |
| properties.numberOfMachines   | Number of machines in the project.                               |
| tags                          | Tags provided by Azure Tagging service.                          |

For properties applicable to all resources, such as `type`, `name`, `id`, and `properties`, refer to [`azure_generic_resource`](azure_generic_resource.md#properties).

> Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/migrate/assessment/projects/get) for other properties available. Any attribute in the response may be accessed with the key names separated by dots (`.`).

Reviewer: ```suggestion
Refer to the [Azure documentation](https://docs.microsoft.com/en-us/rest/api/migrate/assessment/projects/get) for other properties available. Access any attribute in the response by separating the key names with a period (`.`).
```

## Examples

> ### Test that the Migrate Assessment Project has a minimum scalingFactor

Reviewer: ```suggestion
### Test that the Migrate assessment project has a minimum scaling factor
```
